docs/dev/install.rst

```diff
@@ -193,4 +193,4 @@ code. This will save time when submitting pull requests.
 If you ever need to bypass Therapist, you can do so by passing
 ``--no-verify`` to your ``git commit`` command.
 
-.. _Therapist: http://therapist.readthedocs.io/en/latest/overview.html
\ No newline at end of file
+.. _Therapist: http://therapist.readthedocs.io/en/latest/overview.html
```

Comment:
Git tells me there is a change here, but I can't tell what it is. Did you change anything here?

Reply:
I suspect my editor makes sure the file has a last final linebreak.